lib/prismic/fragments/image.rb

```diff
@@ -53,7 +53,8 @@ def ratio
 
         def as_html(link_resolver=nil)
           html = []
-          html << (link_to.nil? ? '' : link_to.start_html(link_resolver))
+          target = defined?(link_to.target) ? link_to.target : nil
```

Comment:
In which case target is undefined?

Reply:
And, following the next line, we check link_to isn't nil. Shouldn't do the same here?